hail/src/main/scala/is/hail/scheduler/Client.scala

```diff
@@ -107,7 +107,7 @@ class SchedulerAppClient(host: String) {
     var s = socket
     if (s == null) {
       s = new DataSocket(new Socket(host, 5053))
-      println(s"SchedulerAppClient.getConnection: connected to $host:5053")
+      info(s"SchedulerAppClient.getConnection: connected to $host:5053")
```

Comment:
Do you want info or log.info?

Reply:
I figured this could print to the console since that's what it currently does and I kind of like that. The other messages that I added just go straight to the log.